tuf/repository_tool.py

```diff
@@ -179,6 +179,26 @@ class Repository(object):
       The name of the repository.  If not supplied, 'rolename' is added to the
       'default' repository.
 
+    use_timestamp_length:
+      Used to decide if the length attribute should be used in the timestamp file.
+      From version 1.0.1 of the tuf spec, length is an optional attribute in
+      timestamp metadata file. Default is true.
+
+    use_timestamp_hashes:
+      Used to decide if the hashes attribute should be used in the timestamp file.
+      From version 1.0.1, of the tuf spec, hashes is an optional attribute in
+      timestamp metadata file. Default is true.
+
+    use_snapshot_length:
+      Used to decide if the length attribute should be used in the snapshot file.
+      From version 1.0.1 of the tuf spec, length is an optional attribute in
+      snapshot metadata file. Default is true.
+
+    use_snapshot_hashes:
+      Used to decide if the hashes attribute should be used in the snapshot file.
+      From version 1.0.1, of the tuf spec, hashes is an optional attribute in
+      snapshot metadata file. Default is true.
+
```

Comment:
I suggested phrasing changes to the function docstrings above and that we should remove the spec version information. Please ensure you update here also.